### PR TITLE
Enable CoreDNS as default for kube-up deployments

### PR DESCRIPTION
**Pull request by fturib**
**What this PR does / why we need it**:
Enable CoreDNS as default (for kube-up installations)
It will allow to run CI tests to prepare graduation criteria for CoreDNS as Default

see : KEP - https://github.com/kubernetes/community/blob/master/keps/sig-network/0012-20180518-coredns-default-proposal.md
see also : https://github.com/kubernetes/features/issues/566

NOTE for release : I guess that CoreDNS as default server for k8s needs a longer description. This specific PR is to ensure we validate all e2e.

```release-note
Make CoreDNS be the default DNS server in kube-up (instead of kube-dns formerly). 
It is still possible to deploy kube-dns by setting CLUSTER_DNS_CORE_DNS=false.
```

### CODE REVIEW

**Comment by fturib** [comment]
@rajansandeep : I spend long time to check the failing e2e tests for the other PR. And I guess there cases when the e2e pull-kubernetes-gce ends in cored dumped and then timeout.
That does not explain all the issue  .. but I would like to test with another PR.

Could you set the flat "ok-to-test" ? Thanks!
related PR is here : #62147 


And, for sure, the I cannot reproduce on similar deployment e2e on GCE ... :)
Here is what I get on test on a private test on GCE 👍 

running:
```
cd $GOPATH/src/github.com/kubernetes/kubernetes
kubetest --dump=/tmp/whatever/_artifacts --gcp-service-account=/home/admin/k8s-coredns-integration-8b248dc238e6.json --up --test --provider=gce --gcp-node-image=gci --gcp-zone=us-central1-f --ginkgo-parallel=30 --runtime-config=batch/v2alpha1=true,admissionregistration.k8s.io/v1alpha1=true --timeout=65m --test_args="--ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8"
```

result:
```
Ran 486 of 1032 Specs in 1215.969 seconds
SUCCESS! -- 486 Passed | 0 Failed | 0 Pending | 546 Skipped 
```

**Comment by fturib** [comment]
With consensus on the way to resolve this issue : https://github.com/kubernetes/dns/issues/254

Adding port remapping to 10053.
That will avoid to deal with super-admin privileges and therefore with PodSecurityPolicies.
This mapping was the only point that needed privileges

**Comment by fturib** [comment]
I mad a mistaking in build the push ... and therefore for a few time had a lot of code. and the Bot added a lot of labels.
I change the commit and have now the correct change. However the labels are not going away ...

@rajansandeep : do you have a way to remove the unecessary labels : all "area/..", "sig/xx"
And then will add the "sig/network". ?

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -848,7 +848,7 @@ ENABLE_RESCHEDULER: $(yaml-quote ${ENABLE_RESCHEDULER:-false})
 LOGGING_DESTINATION: $(yaml-quote ${LOGGING_DESTINATION:-})
 ELASTICSEARCH_LOGGING_REPLICAS: $(yaml-quote ${ELASTICSEARCH_LOGGING_REPLICAS:-})
 ENABLE_CLUSTER_DNS: $(yaml-quote ${ENABLE_CLUSTER_DNS:-false})
-CLUSTER_DNS_CORE_DNS: $(yaml-quote ${CLUSTER_DNS_CORE_DNS:-false})
+CLUSTER_DNS_CORE_DNS: $(yaml-quote ${CLUSTER_DNS_CORE_DNS:-true})
```
[the remark this answers is not in the document]
Comment (fturib) [reply cluster/gce/util.sh]: why ?

**Comment by fturib** [comment]
@johnbelamaric , @thockin , @chrisohaver  : This is the PR for CoreDNS as default for kube-up.

**Comment by rajansandeep** [comment]
@fturib plz correct the release notes

**Comment by thockin** [comment]
@kubernetes/sig-contributor-experience-bugs 

Hey contribex folks - Am I not able to LGTM something without approving?  I see those two statements as very orthogonal.

**Comment by dims** [comment]
@thockin looking at the following 
- https://github.com/kubernetes/test-infra/pull/7504
- https://groups.google.com/forum/#!msg/kubernetes-dev/PJJxV9roXI8/aIeHQdKoBwAJ
- https://github.com/kubernetes/test-infra/commit/9a805e3971f56be35531f19667195b58437889fe

I can flip the config to set `lgtm_acts_as_approve: false` for k/k

**Comment by fturib** [comment]
@thockin : this PR is ready. 
It enables CoreDNS as the default DNS Service when Kubernetes is deplyed with Kube-up.

**This PR does not depend to any release of CoreDNS and can be applied immediately**

NOTE: I think it would be good to merge asap, so we can evaluate impact on ALL the e2e tests that uses kube-up to deploy a cluster for test.

I would not expect issue ... but who knows ?

**Comment by rajansandeep** [comment]
@kubernetes/sig-network-pr-reviews @thockin 
This PR requires the milestone tags and lgtm to merge.

**Pull request merged**